fe/fe-core/src/main/java/org/apache/doris/nereids/util/Utils.java

```diff
@@ -119,4 +120,30 @@ public static List<SlotReference> getOutputSlotReference(Plan plan) {
         return plan.getOutput().stream().map(SlotReference.class::cast)
                 .collect(Collectors.toList());
     }
+
+    /**
+     * Get sql string for plan.
+     *
+     * @param planName name of plan, like LogicalJoin.
+     * @param variables variable needed to add into sqlString.
+     * @return the string of PlanNode.
+     */
+    public static String toSqlString(String planName, String... variables) {
```

Comment:
`Utils.toSqlString('someField', someField.toString()` maybe cause NPE because someField is null.
So I suggest judge variables whether is null at here and convert to 'null' string.
Let's relax to invoke `Util.toSqlString('someField', someField)` and don't care about whether the someField is null.

```suggestion
    public static String toSqlString(String planName, Object... variables) {
```

Reply:
make sense.